src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -122,11 +122,11 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Ranks a series of inputs based on their relevance, training a decision tree ranking model with the <see cref="FastTreeRankingTrainer"/>.
+        /// Create a <see cref="FastTreeRankingTrainer"/>, which ranks a series of inputs based on their relevance and training a decision tree ranking model.
```

Comment:
Suggest: which ranks a series of inputs based on their relevance, using a decision tree ranking model